docs/source/sentiment_tuning_peft.mdx

```diff
@@ -36,6 +36,24 @@ accelerate config # will prompt you to define the training configuration
 accelerate launch scripts/gpt2-sentiment_peft.py # launches training
 ```
 
+## Using `trl` + `peft` and Data Parallelism
+
+You can scale up to as many GPUs as you want, as long as you are able to fit the training process in a single device. The only tweak you need to apply is to load the model as follows:
+```python
+from accelerate import Accelerator
+...
+
+current_device = Accelerator().process_index
+
+pretrained_model = AutoModelForCausalLM.from_pretrained(
+    config.model_name, load_in_8bit=True, device_map={"": current_device}
```

Comment:
Using an empty key in `device_map` seems a bit like magic to me - could we have a one-liner to explain why?

Reply:
Sure yes, I will add it